### PR TITLE
Create a script to load videos from youtube API

### LINKED ISSUE #627: Create a script to load videos from youtube API
**Is your feature request related to a problem? Please describe.**
- The dummy video archive page needs to be replaced with data.

**Describe the solution you'd like**
- Write a script to load items from the url.
- Refer to how mentors profiles are loading in scholarX to get a basic idea.
- Use local copy of this https://sef-dataholder.herokuapp.com/past-one-lives

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #627 

## Goals

- Create a script to load videos from youtube API

## Approach

- Write a script to load items from the url.

### Preview Link

https://pr-628-sef-site.surge.sh/onelive/archive

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/onelive/archive/index.html [hunk 1]
```diff
@@ -154,6 +77,54 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
...
+</script>
+
+<script type="text/html"
+        id="template-youtube-vidoe">
+    <div class="row justify-content-center">
+        <div class="col-9 text-center">
+            <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+            <p class="display-3 text-onelive font-weight-bolder">One Lives</p>
+        </div>
+    </div>
```
Comment (Gravewalker666) [inline src/main/webapp/onelive/archive/index.html]: You can put static parts like this in the body itself. Let's keep only the dynamic parts on the template.

File: src/main/webapp/onelive/index.html [hunk 2]
```diff
@@ -77,8 +77,7 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
             </div>
             <div class="row justify-content-center mb-md-5">
                 <div class="col-9 text-center">
-                    <a href="https://www.facebook.com/pg/sustainableeducationfoundation/events/?ref=page_internal"
-                       target="_blank" class="btn btn-onelive text-uppercase">See upcoming events</a>
+                    <a href="archive/" class="btn btn-onelive text-uppercase">See More videos</a>
```
Comment (Gravewalker666) [inline src/main/webapp/onelive/index.html:80]: See more past onelives would be a better fit of text

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: src/main/webapp/onelive/archive/index.html (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/onelive/archive/index.html]: Ok

**Review by Gravewalker666 — APPROVED** [review]
Good Job! @anjula-sack

File: src/main/webapp/onelive/archive/index.html [hunk 3]
```diff
@@ -154,6 +89,43 @@ <h5 class="text-uppercase">Future-proof - what to learn for the future and how?<
...
+    //function to load youtube videos
+    function loadYoutubeVideos() {
+        $.ajax({
+            type: 'get',
+            url: 'https://sef-dataholder.herokuapp.com/youtube/onelive',
+            dataType: 'json',
+            success: function (data) {
+                //mustache render
+                let renderedData = Mustache.render(
+                    $('#template-youtube-vidoe').html(), {data: data});
```
Comment (jayasanka-sack) [inline src/main/webapp/onelive/archive/index.html:104]: "vidoe"

@Gravewalker666  @anjula-sack

**Comment by jayasanka-sack** [comment]
@anjula-sack @Gravewalker666  Need to do something for long descriptions.

**Comment by Gravewalker666** [comment]
@jayasanka-sack Sorry? I didn't get it.

File: src/main/webapp/onelive/archive/index.html (hunk 3 above)
Comment (Gravewalker666) [reply src/main/webapp/onelive/archive/index.html:104]: Oh! my fault. I must have missed that when I reviewed the code. Sorry for the mistake.

**Pull request merged**